azure-pipelines.yml

```diff
@@ -54,19 +54,25 @@ jobs:
   # just pulling in gfortran
   - script: brew install gcc
     displayName: 'make gfortran available on mac os vm'
+  # we currently seem to use homebrew for openblas in wheels
+  # infrastructure; at the time of writing this means
+  # openblas 0.3.3 on OSX is used vs. Linux clearly targeted at 0.3.0
+  - script: |
+      # NOTE: `brew install --HEAD openblas` for development version
+      # but seems problematic locally
+      brew install openblas
+      # the brew linking command used for wheel builds is a no-op
+      # locally and in Azure, so link openblas manually
+      ln -s /usr/local/opt/openblas/lib/* /usr/local/lib/
+      ln -s /usr/local/opt/openblas/include/* /usr/local/include/
+    displayName: 'install openblas from Homebrew'
   - script: python -m pip install --upgrade pip setuptools wheel
     displayName: 'Install tools'
   - script: python -m pip install cython nose pytz pytest pickle5 vulture
     displayName: 'Install dependencies; some are optional to avoid test skips'
   - script: /bin/bash -c "! vulture . --min-confidence 100 --exclude doc/,numpy/distutils/ | grep 'unreachable'"
     displayName: 'Check for unreachable code paths in Python modules'
-  # NOTE: init_dgelsd failed init issue with current ACCELERATE /
-  # LAPACK configuration on Azure macos image; at the time of writing
-  # this plagues homebrew / macports NumPy builds, but we will
-  # circumvent for now by aggressively disabling acceleration for
-  # macos NumPy builds / tests; ACCELERATE=None on its own is not
```

Comment:
We haven't discussed dropping Accelerate support, but I'd be fine doing that now or later on (following SciPy). Especially if we don't use Accelerate in CI.

Reply:
I'll mark this as resolved -- I can't remember the status of Accelerate deprecation, but it isn't directly connected to using OpenBLAS vs. no acceleration at all here, which is obviously an improvement.